qetpy/core/didv/_plot_didv.py

```diff
@@ -131,14 +143,27 @@ def _plot_time_domain(self, poles, lp_cutoff=None, didv_freq_filt=False):
                 self._dutycycle,
                 **self._1poleresult[key],
             )
-            ax.plot(
-                (self._time + self._1poleresult['params']['dt']) * 1e6,
-                didvfit1_timedomain * 1e6,
-                color='magenta',
-                alpha=0.9,
-                label='1-Pole Fit',
-            )
-
+            
+            if lp_cutoff is not None:
+                lp_pole = lowpassfilter(didvfit1_timedomain,
```

Comment:
perhaps name this lp_1poleresult ? lp_pole makes it sound like the frequency at which the low pass filter is